packages/openapi-v3/src/extension-point/component.ts

```diff
@@ -0,0 +1,12 @@
+import {createBindingFromClass} from '@loopback/context';
+import {Component} from '@loopback/core';
+import {SPEC_SERVICE} from './keys';
+import {SpecService} from './spec-contributor.service';
+
+export class SpecComponent implements Component {
```

Comment:
Maybe it's overkill to have a `SpecComponent`. We can register the `SpecService` as part of `RestComponent`.

Reply:
Good point agreed.